packages/react/src/orders/hooks/__tests__/useOrderReturnOptions.test.ts

```diff
@@ -0,0 +1,343 @@
+import { cleanup, renderHook } from '@testing-library/react';
+import {
+  fetchOrderReturnOptions,
+  resetOrderReturnOptions,
+} from '@farfetch/blackout-redux';
+import {
+  merchantId2,
+  mockState,
+  orderEntityDenormalized,
+  orderId,
+} from 'tests/__fixtures__/orders/orders.fixtures';
+import { useOrderReturnOptions } from '../../..';
+import { withStore } from '../../../../tests/helpers';
+import type { BlackoutError } from '@farfetch/blackout-client';
+
+jest.mock('@farfetch/blackout-redux', () => {
+  const original = jest.requireActual('@farfetch/blackout-redux');
+
+  return {
+    ...original,
+    fetchOrderReturnOptions: jest.fn(() => ({
+      type: 'fetch_order_return_options',
+    })),
+    resetOrderReturnOptions: jest.fn(() => ({
+      type: 'reset_order_return_options',
+    })),
+  };
+});
+
+const defaultReturn = {
+  data: undefined,
+  isLoading: false,
+  isFetched: false,
+  error: null,
+  actions: {
+    fetch: expect.any(Function),
+    reset: expect.any(Function),
+  },
+};
+
+const mockInitialStateNoData = {
+  ...mockState,
+  entities: {
+    ...mockState.entities,
+    orders: {},
+  },
+};
+
+const mockErrorState = {
+  ...mockState,
+  orders: {
+    ...mockState.orders,
+    orderReturnOptions: {
+      ...mockState.orders.orderReturnOptions,
+      error: {
+        [orderId]: new Error('dummy error') as BlackoutError,
```

Comment:
Why not use `toBlackoutError(new Error('dummy error'))` instead of casting?

Reply:
Changed it, when you can, please review. Changed in all files that cast an `Error` to a `BlackoutError` to use `toBlackoutError` function instead.